src/server/troubleshoot/index.md

```diff
@@ -41,4 +41,13 @@ Logs should contain information about sending emails with task `mergin.celery.se
 
 ```shell
 [2024-12-09 10:37:16,265: INFO/ForkPoolWorker-2] Task mergin.celery.send_email_async[3e50df69-90c1-49be-b31c-78f1fb417500] succeeded in 0.11469305199989321s: None
-```
\ No newline at end of file
+```
+
+## Celery settings
```

Comment:
Why do we have these in troubleshooting section? Should not we at least mention few most common issues?
Also there is flask cmd to test if celery is working, worth mentioning.

Reply:
basic settings we can have in 'install' section or with other env variables - ideally our default should be reasonable so users do not need to change anything (unless they know what they are doing)

but because of common issues with celery we should have some trouble shooting section as well - at least to verify that celery is running and accepting tasks